Gruntfile.js

```diff
@@ -74,5 +88,12 @@ module.exports = function(grunt) {
   }
 
   grunt.registerTask("dev", ["connect", "watch"]);
-  grunt.registerTask("test", ["connect", "saucelabs-custom"]);
+  grunt.registerTask("test-locally", ["connect", "jasmine"]);
+
+  var secureEnvVars = process.env.TRAVIS_SECURE_ENV_VARS;
+  if (secureEnvVars === undefined || secureEnvVars === "true") {
```

Comment:
OK, I think I understood. We run with saucelabs if secure env vars are set or if npm test was run outside Travis altogether. Given that only I currently have the SauceLabs credentials, it's safe to only run tests with saucelabs when `secureEnvVars === "true"`, which would simplify the condition.